content/en/docs/collector/configuration.md

```diff
@@ -5,131 +5,317 @@ weight: 20
 
 ## Basics
 
-The Collector handles data via pipelines which are enabled within the
-[service](#service) section. Pipelines are constructed from components that
-access the telemetry data. These components are categorized and configured as:
+The Collector consists of three components that access telemetry data:
 
-* [Receivers](#receivers)
-* [Processors](#processors)
-* [Exporters](#exporters)
+- [Receivers](#receivers)
+- [Processors](#processors)
+- [Exporters](#exporters)
+
+These components once configured must be enabled via pipelines within the
+[service](#service) section.
 
 Secondarily, there are extensions, which provide capabilities that can be added
 to the Collector, but which do not require direct access to telemetry data and
-are not part of pipelines. They are also enabled within the [service](#service) section.
+are not part of pipelines. They are also enabled within the [service](#service)
+section.
 
-## Receivers
+An example configuration would look like:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+processors:
+  batch:
```

Comment:
Is batch necessary with OTLP?

Reply:
In general, using the batch processor is a good idea -- even if the client libraries are batching, it can be good to batch across applications. If client libraries are not configured for batching, then definitely.

Reply:
cool, thanks for the clarification!